config/settings.yml

```diff
@@ -0,0 +1,46 @@
+---
+:ems:
+  :ems_amazon:
+    :disabled_regions: []
+    # disable regions by their keys found in app/models/manageiq/providers/amazon/regions.rb e.g.
+    # uncomment the following to disable gov-west-1 region
```

Comment:
Yeah I was just thinking docs.  That seems the most appropriate place to document new settings.  You may be able to contribute directly in https://github.com/ManageIQ/manageiq_docs, particularly if they already have section somewhere for Advanced Settings.